src/terra-front/modules/Visualizer/LayersTree/LayersTreeItem/LayersTreeItemOptions/LayersTreeItemOptionsDesktop.js

```diff
@@ -1,15 +1,14 @@
-import React from 'react';
-import classnames from 'classnames';
 import { Button, Dialog } from '@blueprintjs/core';
-import LocateButton from '../LocateButton';
-
-import translateMock from '../../../../../utils/translate';
+import classnames from 'classnames';
+import React from 'react';
```

Comment:
Please try not changing order of import when not required to avoid not meaningful changes